editor/block-switcher/index.js

```diff
@@ -90,6 +90,7 @@ class BlockSwitcher extends Component {
 						tabIndex="0"
 						aria-label={ __( 'Block types' ) }
 					>
+						<span>{ __( 'Transform into:' ) }</span>
```

Comment:
Should we use a classname to avoid targeting elements in styling? something like `editor-block-switcher__menu-title`